circle.yml

```diff
@@ -9,5 +9,5 @@ dependencies:
 test:
     override:
         - ! gofmt -s -d . 2>&1 | read
-        - gometalinter ./...
+        - gometalinter --cyclo-over=12 ./...
```

Comment:
The lower, the better. But at some point we cross a threshold that doesn't make sense anymore. I found this value through quite extensive experimentation.